src/Core/OrganizationFeatures/AuthorizationHandlers/GroupOperations.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Microsoft.AspNetCore.Authorization.Infrastructure;
+
+namespace Bit.Core.OrganizationFeatures.AuthorizationHandlers;
+
+public class GroupOperationRequirement : OperationAuthorizationRequirement { }
+
+public static class GroupOperations
+{
+    // Operations on the Group object itself
+    public static readonly GroupOperationRequirement Create = new() { Name = "Create" };
```

Comment:
Little fun thing you can do if you want is `Name = nameof(Create)` then you have one less string literal and it would change name if you ever wanted to refactor the member name.